build_defs/python.build_defs

```diff
@@ -322,6 +322,165 @@ def python_test(name:str, srcs:list, data:list|dict=[], resources:list=[], deps:
         tools=[CONFIG.JARCAT_TOOL],
     )
 
+def python_wheel(name:str, version:str, hashes:list=None, package_name:str=None, outs:list=None,
+                 post_install_commands:list=None, patch:str|list=None, licences:list=None,
+                 test_only:bool&testonly=False, repo:str=None, zip_safe:bool=True, visibility:list=None,
+                 deps:list=[], name_scheme:str=None, strip:list=['*.pyc', 'tests'], tool:str=None):
```

Comment:
We can probably default `tool = CONFIG.PYTHON.WHEEL_TOOL` or similar when we figure out how to distribute it.